src/cpu/jit_uni_resize_bilinear.cpp

```diff
@@ -0,0 +1,161 @@
+/*******************************************************************************
+* Copyright 2018 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#ifdef __INTEL_COMPILER
+#include <immintrin.h>
```

Comment:
Use of intrinsics in the implementation is a non-starter as it introduces additional dependencies:
* specific ISA support is required in the toolchain (compiler and binutils version)
* syntax is not portable between compilers

The tool for low level optimizations Intel MKL-DNN uses is Xbyak.